pegasus/sites.v3/code.org/views/stats_advocate_locally.haml

```diff
@@ -0,0 +1,15 @@
+%link{:href=>"/css/advocate-locally.css", :rel=>"stylesheet"}
+
+.join-coalition-wrapper
+  %img.left-side-image{src:"/images/promote/2018_HoC-463.jpg"}
```

Comment:
We should give this image a simpler and more relevant name.  

Also, in the screenshot the image is squished.. you can use `fit-/fill-` in the URL to crop it.  (See https://github.com/code-dot-org/code-dot-org/blob/8990f16a29d8a1cf9dba0ffb220bd424e8542810/lib/cdo/pegasus/graphics.rb)